sdks/java/core/src/main/java/org/apache/beam/sdk/util/HistogramData.java

```diff
@@ -77,6 +78,54 @@ public static HistogramData linear(double start, double width, int numBuckets) {
     return new HistogramData(LinearBuckets.of(start, width, numBuckets));
   }
 
+  /**
+   * Returns a histogram object wiht exponential boundaries. The input parameter {@code scale}
+   * determines a coefficient 'base' which species bucket boundaries.
+   *
+   * <pre>
+   * base = 2**(2**(-scale)) e.g.
+   * scale=1 => base=2**(1/2)=sqrt(2)
+   * scale=0 => base=2**(1)=2
+   * scale=-1 => base=2**(2)=4
+   * </pre>
+   *
+   * This bucketing strategy makes it simple/numerically stable to compute bucket indexes for
+   * datapoints.
+   *
+   * <pre>
+   * Bucket boundaries are given by the following table where n=numBuckets.
+   * | 'Bucket Index' | Bucket Boundaries   |
+   * |---------------|---------------------|
+   * | Underflow     | (-inf, 0)           |
+   * | 0             | [0, base)           |
+   * | 1             | [base, base^2)      |
+   * | 2             | [base^2, base^3)    |
+   * | i             | [base^i, base^(i+1))|
+   * | n-1           | [base^(n-1), base^n)|
+   * | Overflow      | [base^n, inf)       |
+   * </pre>
+   *
+   * <pre>
+   * Example sacle/boundaries:
+   * When scale=1, buckets 0,1,2...i have lowerbounds 0, 2^(1/2), 2^(2/2), ... 2^(i/2).
+   * When scale=0, buckets 0,1,2...i have lowerbounds 0, 2, 2^2, ... 2^(i).
+   * When scale=-1, buckets 0,1,2...i have lowerbounds 0, 4, 4^2, ... 4^(i).
+   * </pre>
+   *
+   * Scale parameter is similar to OpenTelemetry's notion of ExponentialHistogram.
```

Comment:
`<a href="https://opentelemetry.io/docs/specs/otel/metrics/data-model/#exponentialhistogram">OpenTelemetry's notion of ExponentialHistogram</a>`

Reply:
Done.